frame/ethereum/src/lib.rs

```diff
@@ -72,6 +85,10 @@ pub trait Trait: frame_system::Trait<Hash=H256> + pallet_balances::Trait + palle
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Ethereum {
+		/// The current version of the storage schema. This is useful for tracking and executing storage migrations
+		/// as well as querying storage data from optimized RPC handlers
+		Schema: EthereumStorageSchema;
```

Comment:
Is this PR actually ready? Where is the schema being used right now?

Reply:
I still have to make the tests pass. But it is ready enough that you can see how it works. This storage value is read by the `EthApi` through the `StorageProvider` trait.

The storage key is constructed here https://github.com/paritytech/frontier/pull/199/files#diff-d5674a04eba0110e1cda44150baf38a6f2fab62561ee2033e2066a537bf97b69R54-R62

And the schema version is read in this helper function https://github.com/paritytech/frontier/pull/199/files#diff-d5674a04eba0110e1cda44150baf38a6f2fab62561ee2033e2066a537bf97b69R265-R270